<div class="container mat-elevation-z2 px-0">
    <div class="container" style="background-color: white; padding-bottom: 0.5rem; white-space: pre-line">
        <h4 style="padding-top: 0.5rem">Help - Glossary</h4>
        <section id="Glossary">
<!--            <h5>Glossary</h5>-->
            <dl class="px-3">
                <dt>Coordinate system</dt>
                <dd>Genome positions of ASBs and SNVs
                    in the database are vcf/gtf-like (1-based).<br></dd>
                <dt>BAD</dt>
                <dd>Background Allelic Dosage (BAD) is the expected ratio
                    of major to minor allelic frequencies in a particular genomic region.
                    <br><br>
                    For example, if a the same copy number of two alternating alleles
                    is the same,  alleles,  (e.g. 1:1 (diploid), 2:2, or 3:3,) then
                    the respective region has BAD=1, i.e. the expected ratio of
                    reads mapped to alternative alleles on heterozygous SNVs is 1.
                    All triploid regions have BAD=2 and the expected allelic read
                    ratio is either 2 or Â½. In general, if BAD of a particular region
                    is known, then the expected frequencies of allelic reads
                    are 1/(BAD +1) and BAD/(BAD + 1).
                    <br>
                </dd>
            </dl>
        </section>
        <section
            id="ASB_properties"
        >
            <h5>ASB properties</h5>
            <dl class="px-3">
                <dt>ASB significance</dt>
                <dd>ASB calling is done separately for each ChIP-Seq experiment.
                    For each candidate ASB site, the P-values for Reference and
                    Alternative allele are calculated separately according to the
                    fitted Negative Binomial Mixture model.
                    <br><br>
                    For a particular SNV, the The P-values from individual data sets
                    are aggregated for each TF (for ChIP-Seq data from all cell types)
                    and cell types (for ChIP-Seq data from all TFs). The aggregated
                    P-values are then corrected for multiple tested SNPs using
                    Benjamini-Hochberg (FDR) procedure.
                    <br>
                </dd>
                <dt>ASB effect size</dt>
                <dd>The Effect Size of ASB is calculated separately for Reference
                    and Alternative alleles and is defined as the weighted mean of
                    log-ratios of observed and expected allelic read counts,
                    with weights being -log10 of the respective P-values.
                    The expected read counts are estimated from the fitted
                    Negative Binomial Mixture model.
                    <br><br>
                    The Effect Size is not assigned (n/a) if all of the raw
                    individual P-values of an SNV on a particular genome position
                    are equal to 1, considering Ref- and Alt-ASBs separately.
                    <br>
                </dd>
            </dl>
        </section>
        <section id="Motif_annotation">
            <h5>Motif annotation</h5>
            <dl class="px-3">
                <dt>Motif P-value</dt>
                <dd>For transcription factors with motifs available in
                    the <a
                        rel="noopener noreferrer" target="_blank"
                        href="https://hocomoco11.autosome.ru/"
                    >
                        HOCOMOCO v.11
                    </a>
                    core collection, the P-values of the best hits were
                    calculated for Reference and Alternative allelic variants
                    using <a
                        rel="noopener noreferrer" target="_blank"
                        href="https://github.com/autosome-ru/sarus"
                    >SPRY-SARUS</a>.
                    The motif position was fixed  according to the best hit considering
                    both the Reference and the Alternative alleles on both DNA strands.

                    <br>
                </dd>
                <dt>Motif Fold Change</dt>
                <dd>Motif Fold Change is the log2-ratio between Reference and Alternative Motif P-values.
                    Positive values indicate Alt-ASBs
                    (preferred binding to the Alternative allele).
                    Negative values indicate Ref-ASBs. The value is not assigned
                    (n/a) in case the sequence motif model was not available.
                </dd>
                <div id="Motif_Concordance">
                    <b>Motif Concordance</b><br>
                    <span>
                    Motif Concordance indicates whether the allelic read
                    imbalance is consistent with the transcription factor
                    motif Fold Change (FC, predicted from sequence analysis).
                    The following notation is used:
                    </span>
                    <ul style="list-style-type: circle;">
                        <li>
                            <b>n/a:</b>
                            Motif is not available</li>
                        <li>
                            <b>No hit:</b>
                            The best hit P-value is higher than 0.0005 threshold
                        </li>
                        <li>
                            <b>Weak concordant:</b>
                            The absolute value of FC is less than 2 but
                            consistent with the allelic read imbalance
                        </li>
                        <li>
                            <b>Weak discordant:</b>
                            The absolute value of FC is less than 2 and
                            not consistent with the allelic read imbalance
                        </li>
                        <li>
                            <b>Concordant:</b>
                            The absolute value of FC is greater or equal
                            than 2 and consistent with allelic read imbalance
                        </li>
                        <li>
                            <b>Discordant:</b>
                            The absolute value of  FC is greater or equal than
                            2 but not consistent with allelic read imbalance
                        </li>
                    </ul>
                </div>
            </dl>
        </section>
        <section id="API">
            <h5>API</h5>
            <div class="px-3">
                <span>For programmatic access,
                    ADASTRA API is available at <a href="https://adastra.autosome.ru/api/v1/" target="_blank" rel="noopener noreferrer">https://adastra.autosome.ru/api/v1/</a></span>
            </div>
        </section>
    </div>
</div>
