<div class="container mat-elevation-z2 px-0">
    <div class="container px-0" style="background-color: white; white-space: pre-line">
        <h4 style="padding-top: 0.5rem; padding-left: 15px">Help</h4>
        <mat-accordion multi displayMode="flat">
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">
                            What is ASB
                        </h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <div>
                    <b>ASB</b> stands for <b>A</b>llele <b>S</b>pecific <b>B</b>inding -
                    an event of a particular TF ... . ANANASTRA annotates provided SNPs with ASBs
                    from <a href="https://adastra.autosome.ru"
                            rel="noopener noreferrer" target="_blank"
                >
                    ADASTRA</a> database based on more than 15000
                    <a href="https://en.wikipedia.org/wiki/ChIP_sequencing"
                       rel="noopener noreferrer"
                       target="_blank"
                    >
                        ChIP-seq</a>
                    experiments from
                    <a href="http://gtrd.biouml.org/"
                       rel="noopener noreferrer"
                       target="_blank"
                    >
                        GTRD</a>.
                </div>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="padding-top: 0.5rem">Glossary</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <section id="Glossary">
                    <dl class="px-3">
                        <dt>Coordinate system</dt>
                        <dd>Genome positions of ASBs and SNVs
                            in the database are vcf/gtf-like (1-based), hg38 genome assembly.<br></dd>
                        <dt>BAD</dt>
                        <dd>Background Allelic Dosage (BAD) is the expected ratio
                            of major to minor allelic frequencies in a particular genomic region.
                            <br>
                            For example, if a the same copy number of two alternating alleles
                            is the same,  alleles,  (e.g. 1:1 (diploid), 2:2, or 3:3,) then
                            the respective region has BAD=1, i.e. the expected ratio of
                            reads mapped to alternative alleles on heterozygous SNVs is 1.
                            All triploid regions have BAD=2 and the expected allelic read
                            ratio is either 2 or Â½. In general, if BAD of a particular region
                            is known, then the expected frequencies of allelic reads
                            are 1/(BAD +1) and BAD/(BAD + 1).
                            <br>
                        </dd>
                    </dl>
                </section>
                <section
                    id="ASB_properties"
                >
                    <h5>ASB properties</h5>
                    <dl class="px-3">
                        <dt>ASB significance</dt>
                        <dd>ASB calling is done separately for each ChIP-Seq experiment.
                            For each candidate ASB site, the P-values for Reference and
                            Alternative allele are calculated separately according to the
                            fitted Negative Binomial Mixture model.
                            <br>
                            For a particular SNV, the P-values from individual data sets
                            are aggregated for each <b>T</b>ranscription <b>F</b>actor (for ChIP-Seq data from all cell types),
                            providing <b>TF-ASBs</b>
                            and cell types (for ChIP-Seq data from all TFs), providing <b>cell type-ASBs</b>. The aggregated
                            P-values are then corrected for multiple tested SNPs using
                            Benjamini-Hochberg (FDR) procedure.
                            <br>
                        </dd>
                        <dt>ASB effect size</dt>
                        <dd>The Effect Size of ASB is calculated separately for Reference
                            and Alternative alleles and is defined as the weighted mean of
                            log-ratios of observed and expected allelic read counts,
                            with weights being -log10 of the respective P-values.
                            The expected read counts are estimated from the fitted
                            Negative Binomial Mixture model.
                            <br>
                            The Effect Size is not assigned (n/a) if all of the raw
                            individual P-values of an SNV on a particular genome position
                            are equal to 1, considering Ref- and Alt-ASBs separately.
                            <br>
                        </dd>
                        <dt>ASB SNPs</dt>
                        <dd>If there is at least one (TF- or cell type-) ASB event at SNP,
                            this SNP is called an <b>ASB SNP</b>.
                            <br>
                        </dd>
                    </dl>
                </section>
                <section id="Motif_annotation">
                    <h5>Motif annotation</h5>
                    <dl class="px-3">
                        <div id="Motif_Concordance">
                            <b>Motif Concordance</b><br>
                            <span>
                        Motif Concordance indicates whether the allelic read
                        imbalance is consistent with the transcription factor
                        motif Fold Change (FC, predicted from sequence analysis).
                        The following notation is used:
                        </span>
                            <ul style="list-style-type: circle;">
                                <li>
                                    <b>n/a:</b>
                                    Motif is not available</li>
                                <li>
                                    <b>No hit:</b>
                                    The best hit P-value is higher than 0.0005 threshold
                                </li>
                                <li>
                                    <b>Weak concordant:</b>
                                    The absolute value of FC is less than 2 but
                                    consistent with the allelic read imbalance
                                </li>
                                <li>
                                    <b>Weak discordant:</b>
                                    The absolute value of FC is less than 2 and
                                    not consistent with the allelic read imbalance
                                </li>
                                <li>
                                    <b>Concordant:</b>
                                    The absolute value of FC is greater or equal
                                    than 2 and consistent with allelic read imbalance
                                </li>
                                <li>
                                    <b>Discordant:</b>
                                    The absolute value of  FC is greater or equal than
                                    2 but not consistent with allelic read imbalance
                                </li>
                            </ul>
                        </div>
                    </dl>
                </section>
                <section id="ASB_enrichement">
                    <h5>ASB enrichment</h5>
                    <dl class="px-3">
                        <dt>Overall enrichment</dt>
                        <dd>
                            One-sided Fisher's exact test is performed to assess the
                            ASB enrichment significance. The following contingency table is
                            used
                            <table class="my-table">
                                <tbody>
                                <tr style="height:30px">
                                    <td># of found ASB SNPs</td>
                                    <td># of found non-ASB SNPs</td>
                                </tr>
                                <tr style="height:30px">
                                    <td># of ASB SNPs in database</td>
                                    <td># of non-ASB SNPs in database</td>
                                </tr>
                                </tbody>
                            </table>
                        </dd>
                        <dt>Particular TF or cell type enrichment</dt>
                        <dd>text
                            <br>
                        </dd>
                    </dl>
                </section>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0" expanded>
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">
                            Output specifications</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <div>
                    ANANASTRA report page contains 3 main parts:
                    <h5>Summary</h5>
                    <h5>ASB events of particular TFs or cell types</h5>
                    <h5>SNP annotation with ASB</h5>
<!--                    <dl class="px-3">-->
<!--                        <dt>Summary</dt>-->
<!--                    </dl>-->
                </div>
            </mat-expansion-panel>
        </mat-accordion>
    </div>
</div>
