<div class="container mat-elevation-z2 px-0">
    <div class="container px-0" style="background-color: white; white-space: pre-line">
        <h4 style="padding-top: 0.5rem; padding-left: 15px">Help</h4>
        <mat-accordion multi displayMode="flat">
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">
                            Where to start?</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <div class="px-1">
                    <dl>
                        <dt>Page tour</dt>
                        <dd style="display:flex; align-items: center">
                            <img
                                style="height: 100px; margin-right: 1rem"
                                [src]="'assets/images/example.' + (isAdastra ? 'adastra' : 'ananas') + '.png'"
                                alt=""
                            >
                            <span>
                                <span *ngIf="!isAdastra">The home page and resulting report page</span>
                                <span *ngIf="isAdastra">The home page, search page and SNP page</span>
                            include the 'page tours' in the bottom right corner. Use this icon!</span>
                        </dd>
                        <ng-container *ngIf="!isAdastra">
                            <dt>Examples</dt>
                            <dd>
                                The following examples (1-2) are provided for
                                getting started with ANANASTRA:
                                <br>
                                (1) SNPs associated with Inflammatory bowel disease
                                <a
                                    rel="noopener noreferrer"
                                    target="_blank"
                                    href="https://doi.org/10.1038/nature22969"
                                >
                                    doi:10.1038/nature22969</a>
                                <br>
                                (2) Top 500 significant SNPs found in COVID19-hg GWAS meta-analyses round 6
                                <a
                                    href="https://www.covid19hg.org/results/r6/"
                                    rel="noopener noreferrer"
                                    target="_blank"
                                >
                                    https://www.covid19hg.org/results/r6/</a>, B2: hospitalized covid vs. population
                            </dd>
                        </ng-container>
                    </dl>
                </div>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">
                            What is ASV?
                        </h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <div class="px-1">
                    <b>ASV</b> stands for <b>A</b>llele-<b>S</b>pecific <b>V</b>ariant.
                    One of two alternative alleles of homologous chromosomes may mark a different epigenomic or regulatory state of a genomic region and thus exhibit allele-specific chromatin accessibility.
                    ASV highlights regulatory SNPs with a high potential to affect gene expression.
                    Systematic dissection of ATAC-/DNase-/FAIRE-Seq data processed in the                     <a href="http://gtrd.biouml.org/"
                                                                                                                 rel="noopener noreferrer"
                                                                                                                 target="_blank"
                >
                    GTRD</a> database allowed us to identify hundreds of thousands of ASV events in a wide range of cell types.
                    ASBs are related to ASVs, see also the Help page in
                    <a href="https://adastra.autosome.org/"
                        rel="noopener noreferrer"
                        target="_blank"
                    >
                        ADASTRA</a>.

<!--                    <img src="assets/images/what_is_asA.svg"-->
<!--                         alt=""-->
<!--                         width="100%" style="margin-top: 1rem">-->
                </div>
            </mat-expansion-panel>

            <mat-divider></mat-divider>

            <mat-expansion-panel class="mat-elevation-z0" #glossary>
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="padding-top: 0.5rem">Glossary</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <section id="Glossary">
                    <dl class="px-3">
                        <dt>Coordinate system</dt>
                        <dd>Genome positions of ASVs and SNVs in the online database are vcf/gtf-like (1-based), hg38 genome assembly.
                            The downloads are provided in a bed-like format (0-based start, 1-based end).<br></dd>
                        <dt>BAD</dt>
                        <dd>Background Allelic Dosage (BAD) is the expected ratio
                            of major to minor allelic frequencies in a particular genomic region.
                            <br>
                            For example, if a copy number of two alternating alleles
                            is the same (e.g. 1:1 (diploid), 2:2, or 3:3), then
                            the respective region has BAD=1, i.e. the expected ratio of
                            reads mapped to alternative alleles on heterozygous SNVs is 1.
                            All triploid regions have BAD=2 and the expected allelic read
                            ratio is either 2 or ½. In general, if BAD of a particular region
                            is known, then the expected frequencies of allelic reads
                            are 1/(BAD +1) and BAD/(BAD + 1). Whole genome BADmaps are obtained with
                            <a
                                href="https://github.com/autosome-ru/BABACHI"
                                target="_blank"
                                rel="noopener noreferrer"
                            >BABACHI</a>.
                            More details can be found in the
                            <a
                                href="https://www.nature.com/articles/s41467-021-23007-0"
                                rel="noopener noreferrer" target="_blank"
                            >ADASTRA paper</a>.
                            <br>
                        </dd>
                        <dt id="coverage_filter">ADASTRA coverage filters</dt>
                        <dd>
                            ADASTRA pipeline utilizes several SNV filters before statistical evaluations of ASV:<br>
                            <ul style="list-style-type: circle;">
                                <li>Heterozygous biallelic SNVs (according to GATK ‘GT’ annotation);</li>
                                <li>Having ≥ 5 reads at each of the alleles;</li>
                                <li>Present in dbSNP build151 ‘common’ collection;</li>
                                <li>Belonging to the regions with BABACHI-estimated BAD;</li>
                                <li>Having a total read coverage of ≥ 20 in at least one of the experiments (for a particular TF or cell type).</li>
                            </ul>
                            More details can be found in the
                            <a
                                href="https://www.nature.com/articles/s41467-021-23007-0"
                                rel="noopener noreferrer" target="_blank"
                            >ADASTRA paper</a>.
                            <br>
                        </dd>
                    </dl>
                </section>
                <section
                    id="ASV_properties"
                >
                    <h5 class="px-1">ASV properties</h5>
                    <dl class="px-3">
                        <dt id="asv_significance">ASV significance</dt>
                        <dd>ASV significance
                            ASV calling is done separately for each ChIP-Seq experiment.
                            For each candidate ASV site, the P-values for Reference and
                            Alternative allele are calculated separately according to the
                            fitted Negative Binomial Mixture model accounting for different
                            assignment of the alleles to the higher or lower DNA copies in
                            genomic regions with BAD > 1. Prior mixture weights obtained
                            with the global fit across SNVs were updated with Bayesian
                            estimation separately for each SNV.
                            <br>
                            For a particular SNV, the P-values from individual data sets are
                            aggregated with the logit (Mudholkar-George) method for each TF
                            (using ChIP-Seq data from all cell types) and cell type (using ChIP-Seq data from
                            all TFs) and FDR-corrected with the Benjamini-Hochberg procedure for SNVs
                            (for each TF and each cell type separately).

                            <br>
                        </dd>
                        <dt id="ASV_es"
                        >ASV effect size</dt>
                        <dd>The Effect Size of an ASV event is calculated separately for Reference
                            and Alternative alleles and is defined as the weighted mean of log-ratios
                            of observed and expected allelic read counts, with weights being -log<sub>10</sub>
                            of the respective P-values. The expected read counts are estimated from
                            the fitted Negative Binomial Mixture model accounting for different
                            assignments of the allies to the higher or lower DNA copies in genomic
                            regions with BAD > 1. Prior mixture weights obtained with the global fit
                            across SNVs were updated with Bayesian estimation separately for each SNV.
                            <br>
                            The Effect Size is not assigned (n/a) if all of the raw individual
                            P-values of an SNV on a particular genome position are equal to 1,
                            considering Ref- and Alt-ASVs separately.
                            <br>
                        </dd>
                    </dl>
                </section>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <ng-container *ngIf="isAdastra">
            <mat-expansion-panel  class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">Software</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <section id="API">
                    <div class="px-3">
                        <dl>
                            <dt>API</dt>
                            <dd>For programmatic access,
                                UDACHA API is available at
                                <a
                                    href="/api/"
                                    target="_blank"
                                    rel="noopener noreferrer"
                                >
                                    {{'https://udacha.autosome.org/api/'}}
                                </a>
                            </dd>
                            <dt>BABACHI</dt>
                            <dd>BABACHI is available on
                                <a
                                    href="https://github.com/autosome-ru/BABACHI"
                                    target="_blank"
                                    rel="noopener noreferrer"
                                >
                                    GitHub
                                </a>
                            </dd>
                            <dt>MIXALIME</dt>
                            <dd>MIXALIME is available on
                                <a
                                    href="https://github.com/autosome-ru/MIXALIME"
                                    target="_blank"
                                    rel="noopener noreferrer"
                                >
                                    GitHub
                                </a>
                            </dd>
                        </dl>
                    </div>
                </section>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">External resources</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <section >
                    <h5 id="used-software">Resources and databases used to annotate UDACHA ASVs</h5>
                    <div class="px-3">
                        <ul class="list-circle">
                            <li>
                                <b>
                                    <a
                                        href="https://www.ncbi.nlm.nih.gov/snp/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        dbSNP
                                    </a>
                                </b> build 151
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://www.gencodegenes.org/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        GENCODE</a>
                                </b> v35
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://hocomoco11.autosome.org/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        HOCOMOCO</a>
                                </b> v11
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://www.gtexportal.org/home/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        GTEx</a>
                                </b> release V8
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://www.ncbi.nlm.nih.gov/clinvar/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        ClinVar</a>
                                </b> release 9/05/2019
                            </li>

                            <li>
                                <b>
                                    <a
                                        href="https://www.ebi.ac.uk/gwas/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        NHGRI-EBI GWAS</a>
                                </b> release 8/27/2019 with EFO mappings
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://phewascatalog.org/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        PheWAS catalog</a>
                                </b> release V8
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://grasp.nhlbi.nih.gov/Overview.aspx"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        GRASP</a>
                                </b> build 2.0.0.0
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="http://pubs.broadinstitute.org/pubs/finemapping/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        BROAD fine-mapping</a>
                                </b>
                            </li>
                        </ul>
                    </div>
                </section>
            </mat-expansion-panel>
            </ng-container>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">
                            Browser compatibility</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                Browser compatibility was checked using
                <a
                    href="https://saucelabs.com/"
                    target="_blank"
                    rel="noopener noreferrer"
                >Sauce Labs</a> and
                <a
                    href="https://www.browserstack.com/"
                    target="_blank"
                    rel="noopener noreferrer"
                >BrowserStack</a>.
                <div style="overflow: auto;">
                <table class="my-table">
                    <tr>
                        <th>OS</th>
                        <th>Version</th>
                        <th>Chrome</th>
                        <th>Firefox</th>
                        <th>Microsoft Edge</th>
                        <th>Opera</th>
                        <th>Safari</th>
                    </tr>
                    <tr>
                        <td>Linux</td>
                        <td>Ubuntu 18.04</td>
                        <td>93</td>
                        <td>92</td>
                        <td>n/a</td>
                        <td>not tested</td>
                        <td>n/a</td>
                    </tr>
                    <tr>
                        <td>MacOS</td>
                        <td>Catalina, Mojave</td>
                        <td>93</td>
                        <td>92</td>
                        <td>n/a</td>
                        <td>79</td>
                        <td>11.1</td>
                    </tr>
                    <tr>
                        <td>Windows</td>
                        <td>10</td>
                        <td>93</td>
                        <td>92</td>
                        <td>93</td>
                        <td>79</td>
                        <td>n/a</td>
                    </tr>
                </table>
                </div>
            </mat-expansion-panel>
        </mat-accordion>
    </div>
</div>
<ng-template #preview let-data>
    <h1
        style="display: flex; justify-content: space-between; margin-bottom: 0;"
        mat-dialog-title
    >
        <span>{{getNameOfId(data)}} preview</span>
        <button mat-dialog-close
                mat-icon-button tabindex="-1">
            <mat-icon svgIcon="clear"></mat-icon>
        </button>
    </h1>
    <mat-dialog-content>
        <img
            style="max-width: 100%"
            alt="Loading..."
            [src]="'/assets/images/' + data + '.help.png'"
        >
    </mat-dialog-content>
</ng-template>

<ng-template #concordanceExplanation>

</ng-template>
