<div class="container mat-elevation-z2 px-0">
    <div class="container px-0" style="background-color: white; white-space: pre-line">
        <h4 style="padding-top: 0.5rem; padding-left: 15px">Help</h4>
        <mat-accordion multi displayMode="flat">
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">
                            Where to start?</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <div class="px-1">
                    <dl>
                        <dt>Page tour</dt>
                        <dd style="display:flex; align-items: center">
                            <img
                                style="height: 100px; margin-right: 1rem"
                                [src]="'assets/images/example.' + (isAdastra ? 'adastra' : 'ananas') + '.png'"
                                alt=""
                            >
                            <span>
                            <span *ngIf="!isAdastra">The home page and resulting report page</span>
                            <span *ngIf="isAdastra">The home page, search page and SNP page</span>
                            &nbsp;include the 'page tours'. Use this icon!</span>
                        </dd>
                        <ng-container *ngIf="!isAdastra">
                            <dt>Examples</dt>
                            <dd>
                                The following examples (1-2) are provided for
                                getting started with ANANASTRA:
                                <br>
                                (1) Significant SNPs found in COVID19-hg GWAS meta-analyses round 4
                                <a
                                    href="https://www.covid19hg.org/results"
                                    rel="noopener noreferrer"
                                    target="_blank"
                                >
                                    www.covid19hg.org/results</a>, C1_ALL: Covid vs. lab/self-reported negative;
                                <br>
                                (2) SNPs associated with Inflammatory bowel disease
                                <a
                                    rel="noopener noreferrer"
                                    target="_blank"
                                    href="https://doi.org/10.1038/nature22969"
                                >
                                    doi:10.1038/nature22969</a>.
                            </dd>
                        </ng-container>
                    </dl>
                </div>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">
                            What is ASB?
                        </h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <div class="px-1">
                    <b>ASB</b> stands for <b>A</b>llele-<b>S</b>pecific <b>B</b>inding.
                    A <b>T</b>ranscription <b>F</b>actor (<b>TF</b>) might prefer to bind one of two alternative alleles of
                    homologous chromosomes and thus exhibit allele-specific binding.
                    ASB highlights regulatory SNPs with high potential to affect gene expression.
                    Systematic dissection of ChIP-Seq data processed in the
                    <a href="http://gtrd.biouml.org/"
                       rel="noopener noreferrer"
                       target="_blank"
                    >
                        GTRD</a>
                    database allowed us to identify hundreds of thousands ASB events
                    for a wide range of transcription factors.
                    <span *ngIf="!isAdastra">
                    ANANASTRA annotates a set of user-submitted SNPs with ASBs
                    from the
                    <a href="https://adastra.autosome.ru"
                       rel="noopener noreferrer"
                       target="_blank"
                    >
                        ADASTRA</a> database.</span>
                    <img src="assets/images/what_is_asb.svg"
                         alt=""
                         width="100%" style="margin-top: 1rem">
                </div>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0" #glossary>
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="padding-top: 0.5rem">Glossary</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <section id="Glossary">
                    <dl class="px-3">
                        <dt>Coordinate system</dt>
                        <dd>Genome positions of ASBs and SNVs
                            in the database are vcf/gtf-like (1-based), hg38 genome assembly.<br></dd>
                        <dt>BAD</dt>
                        <dd>Background Allelic Dosage (BAD) is the expected ratio
                            of major to minor allelic frequencies in a particular genomic region.
                            <br>
                            For example, if a copy number of two alternating alleles
                            is the same (e.g. 1:1 (diploid), 2:2, or 3:3), then
                            the respective region has BAD=1, i.e. the expected ratio of
                            reads mapped to alternative alleles on heterozygous SNVs is 1.
                            All triploid regions have BAD=2 and the expected allelic read
                            ratio is either 2 or ½. In general, if BAD of a particular region
                            is known, then the expected frequencies of allelic reads
                            are 1/(BAD +1) and BAD/(BAD + 1). More details can be found in the
                            <a
                                href="https://www.nature.com/articles/s41467-021-23007-0"
                                rel="noopener noreferrer" target="_blank"
                            >ADASTRA paper</a>.
                            <br>
                        </dd>
                        <dt id="coverage_filter">ADASTRA coverage filters</dt>
                        <dd>
                            ADASTRA pipeline utilizes several SNV filters before statistical evaluations of ASB:<br>
                            <ul style="list-style-type: circle;">
                                <li>Heterozygous biallelic SNVs (according to GATK ‘GT’ annotation);</li>
                                <li>Having ≥ 5 reads at each of the alleles;</li>
                                <li>Present in dbSNP build151 ‘common’ collection;</li>
                                <li>Belonging to the regions with BABACHI-estimated BAD;</li>
                                <li>Having a total read coverage of ≥ 20 in at least one of the experiments (for a particular TF or cell type).</li>
                            </ul>
                            More details can be found in the
                            <a
                                href="https://www.nature.com/articles/s41467-021-23007-0"
                                rel="noopener noreferrer" target="_blank"
                            >ADASTRA paper</a>.
                            <br>
                        </dd>
                    </dl>
                </section>
                <section
                    id="ASB_properties"
                >
                    <h5 class="px-1">ASB properties</h5>
                    <dl class="px-3">
                        <dt id="asb_significance">ASB significance</dt>
                        <dd>ASB significance
                            ASB calling is done separately for each ChIP-Seq experiment.
                            For each candidate ASB site, the P-values for Reference and
                            Alternative allele are calculated separately according to the
                            fitted Negative Binomial Mixture model accounting for different
                            assignment of the alleles to the higher or lower DNA copies in
                            genomic regions with BAD > 1. Prior mixture weights obtained
                            with the global fit across SNVs were updated with Bayesian
                            estimation separately for each SNV.
                            <br>
                            For a particular SNV, the P-values from individual data sets are
                            aggregated with the logit (Mudholkar-George) method for each TF
                            (using ChIP-Seq data from all cell types) and cell type (using ChIP-Seq data from
                            all TFs) and FDR-corrected with the Benjamini-Hochberg procedure for SNVs
                            (for each TF and each cell type separately).

                            <br>
                        </dd>
                        <dt id="ASB_es"
                        >ASB effect size</dt>
                        <dd>The Effect Size of an ASB event is calculated separately for Reference
                            and Alternative alleles and is defined as the weighted mean of log-ratios
                            of observed and expected allelic read counts, with weights being -log<sub>10</sub>
                            of the respective P-values. The expected read counts are estimated from
                            the fitted Negative Binomial Mixture model accounting for different
                            assignments of the allies to the higher or lower DNA copies in genomic
                            regions with BAD > 1. Prior mixture weights obtained with the global fit
                            across SNVs were updated with Bayesian estimation separately for each SNV.
                            <br>
                            The Effect Size is not assigned (n/a) if all of the raw individual
                            P-values of an SNV on a particular genome position are equal to 1,
                            considering Ref- and Alt-ASBs separately.
                            <br>
                        </dd>
                    </dl>
                </section>
                <section >
                    <h5 id="motif_annotation">Motif annotation</h5>
                    <dl class="px-3">
                        <dt>Motif P-value</dt>
                        <dd>For transcription factors with motifs available in
                            the <a
                                rel="noopener noreferrer" target="_blank"
                                href="https://hocomoco11.autosome.ru/"
                            >
                                HOCOMOCO v.11
                            </a>
                            core collection, the P-values of the best hits were
                            calculated for Reference and Alternative allelic variants
                            using <a
                                rel="noopener noreferrer" target="_blank"
                                href="https://github.com/autosome-ru/sarus"
                            >SPRY-SARUS</a>.
                            The motif position was fixed  according to the best hit considering
                            both the Reference and the Alternative alleles on both DNA strands.
                            <br>
                        </dd>
                        <dt>Motif Fold Change</dt>
                        <dd>Motif Fold Change is the log2-ratio between Reference and Alternative Motif P-values.
                            Positive values indicate Alt-ASBs
                            (preferred binding to the Alternative allele).
                            Negative values indicate Ref-ASBs. The value is not assigned
                            (n/a) in case the sequence motif model was not available.
                        </dd>
                        <dt id="motif_concordance">
                            Motif Concordance</dt>
                        <dd>
                        Motif Concordance indicates whether the allelic read
                        imbalance is consistent with the transcription factor
                        motif Fold Change (FC, predicted from sequence analysis).
                        The following notation is used:
                            <ul style="list-style-type: circle;">
                                <li>
                                    <b>n/a:</b>
                                    Motif is not available;</li>
                                <li>
                                    <b>No hit:</b>
                                    The best hit P-value is higher than 0.0005 threshold;
                                </li>
                                <li>
                                    <b>Weak concordant:</b>
                                    The absolute value of FC is less than 2 but
                                    consistent with the allelic read imbalance;
                                </li>
                                <li>
                                    <b>Weak discordant:</b>
                                    The absolute value of FC is less than 2 and
                                    not consistent with the allelic read imbalance;
                                </li>
                                <li>
                                    <b>Concordant:</b>
                                    The absolute value of FC is greater or equal
                                    to 2 and consistent with allelic read imbalance;
                                </li>
                                <li>
                                    <b>Discordant:</b>
                                    The absolute value of  FC is greater or equal to
                                    2 but not consistent with allelic read imbalance.
                                </li>
                            </ul>
                        </dd>
                    </dl>
                </section>
                <section  *ngIf="!isAdastra">
                    <h5
                        id="ASB_enrichment"
                    >ASB enrichment estimation</h5>
                    <dl class="px-3">
                        <dt>Overall enrichment for TF- and cell type-specific ASBs</dt>
                        <dd>
                            <div>
                            Right-tailed Fisher's exact test is performed to assess the
                            ASB enrichment with following contingency table:
                            <table class="my-table">
                                <tbody>
                                <tr style="height:30px">
                                    <td># of SNPs with ASBs in user-submitted list</td>
                                    <td># of SNPs with ASBs in the database</td>
                                </tr>
                                <tr style="height:30px">
                                    <td># of non-ASB SNPs* in user-submitted list</td>
                                    <td># of non-ASB SNPs* in the database</td>
                                </tr>
                                </tbody>
                            </table>
                                *Non-ASB SNPs are SNPs without any significant ASBs passing 25% FDR.<br><br>
                                Similar tests are performed to assess enrichment at the
                                level of individual chromosomes (with the individual chromosome P-values
                                being aggregated using the
                                <a href="https://link.springer.com/article/10.1007/BF02056895" rel="noopener noreferrer" target="_blank">‘logit’ method</a>), and for particular
                                TFs or cell types.

                                To have a proper background, non-ASB SNPs are selected
                                to pass the same read coverage thresholds
                                as the genuine significant ASB SNPs, i.e. most of those
                                are located within ChIP-Seq peaks of the respective TFs.
                            </div>
                        </dd>
                        <dt id="background"
                        >
                            Background SNP set
                        </dt>
                        <dd>
                            The following types of background SNP sets are available to assess the overall ASB enrichment:
                            <ul style="list-style-type: circle;">
                                <li>
                                    <b>Whole genome:</b> All ADASTRA 'candidate' SNPs eligible for ASB assessment (see <a (click)="openGlossary('coverage_filter'); $event.preventDefault()"
                                                                                                                          routerLink="/help"
                                                                                                                          fragment="coverage_filter">
                                    here</a>);
                                </li>
                                <li>
                                    <b>LD-islands:</b> ADASTRA 'candidate' SNPs located in the same LD-islands as the ‘test’ SNPs of the user-submitted list;
                                </li>
                                <li>
                                    <b>Local:</b> ADASTRA 'candidate' SNPs located within the same 1 Mbp windows as the ‘test’ SNPs of the user-submitted list.
                                </li>
                            </ul>
                            <b>[!] Note</b>, that chromosome-level and TF-/cell type-specific tests <b>always</b> use the respective subsets of the <b>'whole genome'</b> background set.</dd>
                    </dl>
                </section>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0" *ngIf="!isAdastra">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0" class="d-none d-md-block">
                            Understanding ANANASTRA report pages</h4>
                        <h4 style="margin-bottom: 0" class="d-block d-md-none">
                            Report pages</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <div class="px-3">
                    Each ANANASTRA report page contains 3 tabs:
                    (1) Summary, (2) ASB events of particular TFs, and (3) ASB events found in particular
                    cell types.<br><br>
                    <dl>
                        <dt>Summary</dt>
                        <dd>
                            This section is dedicated to general enrichment on ASB events in the submitted
                            set of SNPs against the chosen background set. We report the observed fraction
                            of ASBs (among ASB-carrying and ASB-less SNPs passing certain read coverage thresholds,
                            see
                            <a (click)="openGlossary('coverage_filter'); $event.preventDefault()"
                                   routerLink="/help"
                                   fragment="coverage_filter">
                                here</a>). ASB enrichment significance is calculated with the right-tailed Fisher's exact
                            test as
                            <a
                                (click)="openGlossary('ASB_enrichment'); $event.preventDefault()"
                                routerLink="/help"
                                fragment="ASB_enrichement"
                            >
                                here</a>.
                            <span
                            class="preview d-none d-lg-inline"
                            (click)="openPreview('summary')"
                        >
                                preview
                        </span><br><br>
                            Below the overal statistics, there is a collapsible table of chromosome-wise ASB enrichment data.
                            <span
                                class="preview d-none d-lg-inline"
                                (click)="openPreview('chromosomes')"
                            >
                                    preview
                            </span>
                        </dd>
                        <dt>
                            ASB events related to particular TFs or cell types
                            </dt>
                        <dd>The second and the third tabs contain information about ASB events of particular
                            TFs or cell types. For each TF and cell type, we compute a separate P-value using
                            the same strategy as for the global enrichment (except the respective background
                            sets are always subsetted from the genome-wide set of 'candidate' SNPs). The results
                            P-values are then corrected for the number of tested TFs / cell types (with at least
                            1 observed ASB in the user's list) using Benjamini-Hogberg (FDR) procedure.
                            The TFs / cell types with the lowest FDR are displayed on top by default.
                            <span
                                class="preview d-none d-lg-inline"
                                (click)="openPreview('enrichment')"
                            >
                                preview</span><br><br>

                            At the bottom of each tab there is a detailed table listing all ASB events
                            (in the expanded view mode) or showing only the single top-significant
                            TF-/cell type- ASB event per SNP (in the collapsed view mode). Each mode
                            allows downloading a complete table (the list of columns visible in the
                            online report can be easily adjusted).
                            <span
                            class="preview d-none d-lg-inline"
                            (click)="openPreview('table')"
                        >
                                preview</span><br><br>
                            <b>[!] Note</b>, that the lists longer than 200 ASBs are not browsable online, but can be downloaded for offline exploration.
                        </dd>
                    </dl>
                </div>
            </mat-expansion-panel>
            <ng-container *ngIf="isAdastra">
            <mat-expansion-panel  class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">Software</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <section id="API">
                    <div class="px-3">
                        <dl>
                            <dt>API</dt>
                            <dd>For programmatic access,
                                ADASTRA API is available at
                                <a
                                    href="https://adastra.autosome.ru/api/"
                                    target="_blank"
                                    rel="noopener noreferrer"
                                >
                                    {{'https://adastra.autosome.ru/api/'}}
                                </a>
                            </dd>
                            <dt>BABACHI</dt>
                            <dd>BABACHI is available on
                                <a
                                    href="https://github.com/autosome-ru/BABACHI"
                                    target="_blank"
                                    rel="noopener noreferrer"
                                >
                                    GitHub
                                </a>
                            </dd>
                            <dt>ADASTRA pipeline</dt>
                            <dd>ADASTRA pipeline is available on
                                <a
                                    href="https://github.com/autosome-ru/ADASTRA-pipeline"
                                    target="_blank"
                                    rel="noopener noreferrer"
                                >
                                    GitHub
                                </a>
                            </dd>
                        </dl>
                    </div>
                </section>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">External resources</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <section >
                    <h5 id="used-software">Resources and databases used to annotate ADASTRA ASBs</h5>
                    <div class="px-3">
                        <ul class="list-circle">
                            <li>
                                <b>
                                    <a
                                        href="https://www.ncbi.nlm.nih.gov/snp/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        dbSNP
                                    </a>
                                </b> build 151
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://www.gencodegenes.org/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        GENCODE</a>
                                </b> v35
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://hocomoco11.autosome.ru/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        HOCOMOCO</a>
                                </b> v11
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://www.gtexportal.org/home/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        GTEx</a>
                                </b> release V8
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://www.ncbi.nlm.nih.gov/clinvar/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        ClinVar</a>
                                </b> release 9/05/2019
                            </li>

                            <li>
                                <b>
                                    <a
                                        href="https://www.ebi.ac.uk/gwas/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        NHGRI-EBI GWAS</a>
                                </b> release 8/27/2019 with EFO mappings
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://phewascatalog.org/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        PheWAS catalog</a>
                                </b> release V8
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="https://grasp.nhlbi.nih.gov/Overview.aspx"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        GRASP</a>
                                </b> build 2.0.0.0
                            </li>
                            <li>
                                <b>
                                    <a
                                        href="http://pubs.broadinstitute.org/pubs/finemapping/"
                                        target="_blank"
                                        rel="noopener noreferrer"
                                    >
                                        BROAD fine-mapping</a>
                                </b>
                            </li>
                        </ul>
                    </div>
                </section>
            </mat-expansion-panel>
            </ng-container>
        </mat-accordion>
    </div>
</div>
<ng-template #preview let-data>
    <h1
        style="display: flex; justify-content: space-between; margin-bottom: 0;"
        mat-dialog-title
    >
        <span>{{getNameOfId(data)}} preview</span>
        <button mat-dialog-close
                mat-icon-button tabindex="-1">
            <mat-icon svgIcon="clear"></mat-icon>
        </button>
    </h1>
    <mat-dialog-content>
        <img
            style="max-width: 100%"
            alt=""
            [src]="'/assets/images/' + data + '.help.png'"
        >
    </mat-dialog-content>
</ng-template>
