<div class="container mat-elevation-z2 px-0">
    <div class="container px-0" style="background-color: white; white-space: pre-line">
        <h4 style="padding-top: 0.5rem; padding-left: 15px">Help</h4>
        <mat-accordion multi displayMode="flat">
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">
                            Where to start?</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <div class="px-3">
                    <dl>
                        <dt>Page tour</dt>
                        <dd style="display:flex; align-items: center">
                            <img
                                style="height: 100px; margin-right: 1rem"
                                src="assets/images/example.ananas.png"
                                alt=""
                            >
                            The home page and resulting report page include the 'page tours'. Use this icon!
                        </dd>
                        <dt>Examples</dt>
                        <dd>
                            The following examples (1-3) are provided for
                            getting started with ANANASTRA:
                            <br>
                            (1) Significant SNPs found in COVID19-hg GWAS meta-analyses round 4
                            <a
                                href="https://www.covid19hg.org/results"
                                rel="noopener noreferrer"
                                target="_blank"
                            >
                                www.covid19hg.org/results</a>, C1_ALL: Covid vs. lab/self-reported negative;
                            <br>
                            (2) Genomic region encompassing NFIA gene encoding Nuclear factor 1 A-type transcription factor;
                            <br>
                            (3) SNPs associated with Inflammatory bowel disease
                            <a
                                rel="noopener noreferrer"
                                target="_blank"
                                href="https://doi.org/10.1038/nature22969"
                            >
                                doi:10.1038/nature22969</a>.
                        </dd>
                    </dl>
                </div>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0">
                            What is ASB
                        </h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <div class="px-3">
                    <b>ASB</b> stands for <b>A</b>llele-<b>S</b>pecific <b>B</b>inding.
                    A transcription factor might prefer to bind one of two alternative alleles of
                    homologous chromosomes and thus exhibit allele-specific binding.
                    ASB highlights regulatory SNPs with high potential to affect gene expression.
                    Systematic dissection of ChIP-Seq data processed in the
                    <a href="http://gtrd.biouml.org/"
                       rel="noopener noreferrer"
                       target="_blank"
                    >
                        GTRD</a>
                    database allowed us to identify hundreds of thousands ASB events
                    for a wide range of transcription factors.

                    ANANASTRA annotates a set of user-submitted SNPs with ASBs
                    from the
                    <a href="https://adastra.autosome.ru"
                       rel="noopener noreferrer"
                       target="_blank"
                    >
                        ADASTRA</a> database.
                </div>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0" #glossary>
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="padding-top: 0.5rem">Glossary</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <section id="Glossary">
                    <dl class="px-3">
                        <dt>Coordinate system</dt>
                        <dd>Genome positions of ASBs and SNVs
                            in the database are vcf/gtf-like (1-based), hg38 genome assembly.<br></dd>
                        <dt>BAD</dt>
                        <dd>Background Allelic Dosage (BAD) is the expected ratio
                            of major to minor allelic frequencies in a particular genomic region.
                            <br>
                            For example, if a copy number of two alternating alleles
                            is the same (e.g. 1:1 (diploid), 2:2, or 3:3), then
                            the respective region has BAD=1, i.e. the expected ratio of
                            reads mapped to alternative alleles on heterozygous SNVs is 1.
                            All triploid regions have BAD=2 and the expected allelic read
                            ratio is either 2 or Â½. In general, if BAD of a particular region
                            is known, then the expected frequencies of allelic reads
                            are 1/(BAD +1) and BAD/(BAD + 1). More details can be found in the
                            <a
                                href="https://www.biorxiv.org/content/10.1101/2020.10.07.327643v3"
                                rel="noopener noreferrer" target="_blank"
                            >ADASTRA preprint</a>.
                            <br>
                        </dd>
                    </dl>
                </section>
                <section
                    id="ASB_properties"
                >
                    <h5>ASB properties</h5>
                    <dl class="px-3">
                        <dt id="sign">ASB significance</dt>
                        <dd>ASB calling is done separately for each ChIP-Seq experiment.
                            For each candidate ASB site, the P-values for Reference and
                            Alternative allele are calculated separately according to the
                            fitted Negative Binomial Mixture model (see Methods in the
                            <a
                                href="https://www.biorxiv.org/content/10.1101/2020.10.07.327643v3"
                                rel="noopener noreferrer" target="_blank"
                            >ADASTRA preprint</a> for details).
                            <br>
                            For a particular SNV, the P-values from individual data sets
                            are aggregated for each <b>T</b>ranscription <b>F</b>actor (for ChIP-Seq data from all cell types),
                            providing <b>TF-ASBs</b>
                            and cell types (for ChIP-Seq data from all TFs), providing <b>cell type-ASBs</b>. The aggregated
                            P-values are then corrected for multiple tested SNPs using
                            Benjamini-Hochberg (FDR) procedure.
                            <br>
                        </dd>
                        <dt>ASB effect size</dt>
                        <dd>The Effect Size of ASB is calculated separately for Reference
                            and Alternative alleles and is defined as the weighted mean of
                            log-ratios of observed and expected allelic read counts,
                            with weights being -log10 of the respective P-values.
                            The expected read counts are estimated from the fitted
                            Negative Binomial Mixture model.
                            <br>
                            The Effect Size is not assigned (n/a) if all of the raw
                            individual P-values of an SNV on a particular genome position
                            are equal to 1, considering Ref- and Alt-ASBs separately.
                            <br>
                        </dd>
                    </dl>
                </section>
                <section id="Motif_annotation">
                    <h5>Motif annotation</h5>
                    <dl class="px-3">
                        <div id="Motif_Concordance">
                            <b>Motif Concordance</b><br>
                            <span>
                        Motif Concordance indicates whether the allelic read
                        imbalance is consistent with the transcription factor
                        motif Fold Change (FC, predicted from sequence analysis).
                        The following notation is used:
                        </span>
                            <ul style="list-style-type: circle;">
                                <li>
                                    <b>n/a:</b>
                                    Motif is not available;</li>
                                <li>
                                    <b>No hit:</b>
                                    The best hit P-value is higher than 0.0005 threshold;
                                </li>
                                <li>
                                    <b>Weak concordant:</b>
                                    The absolute value of FC is less than 2 but
                                    consistent with the allelic read imbalance;
                                </li>
                                <li>
                                    <b>Weak discordant:</b>
                                    The absolute value of FC is less than 2 and
                                    not consistent with the allelic read imbalance;
                                </li>
                                <li>
                                    <b>Concordant:</b>
                                    The absolute value of FC is greater or equal
                                    to 2 and consistent with allelic read imbalance;
                                </li>
                                <li>
                                    <b>Discordant:</b>
                                    The absolute value of  FC is greater or equal to
                                    2 but not consistent with allelic read imbalance.
                                </li>
                            </ul>
                        </div>
                    </dl>
                </section>
                <section id="ASB_enrichement" >
                    <h5
                        [ngClass]="fragment == 'ASB_enrichment' ? 'theme-primary-border' : ''"
                    >ASB enrichment estimation</h5>
                    <dl class="px-3">
                        <dt>Overall enrichment for TF- and cell type-specific ASBs</dt>
                        <dd>
                            <div>
                            Right-sided Fisher's exact test is performed to assess the
                            ASB enrichment significance. The following contingency table is
                            used
                            <table class="my-table">
                                <tbody>
                                <tr style="height:30px">
                                    <td># of SNPs with ASBs in user-submitted list</td>
                                    <td># of SNPs with ASBs in the database</td>
                                </tr>
                                <tr style="height:30px">
                                    <td># of non-ASB SNPs in user-submitted list</td>
                                    <td># of non-ASB SNPs in the database</td>
                                </tr>
                                </tbody>
                            </table>
                            Similar tests are performed to assess individual
                                TF-ASB and cell type-ASB enrichment.
                                To have a proper background, non-ASB SNPs are selected
                                to pass the same read coverage thresholds
                                as the genuine significant ASB SNPs, i.e. most of those
                                are located within ChIP-Seq peaks of the respective TFs.
                            </div>
                        </dd>
                    </dl>
                </section>
            </mat-expansion-panel>
            <mat-divider></mat-divider>
            <mat-expansion-panel class="mat-elevation-z0">
                <mat-expansion-panel-header>
                    <mat-panel-title>
                        <h4 style="margin-bottom: 0" class="d-none d-md-block">
                            Understanding ANANASTRA report pages</h4>
                        <h4 style="margin-bottom: 0" class="d-block d-md-none">
                            Report pages</h4>
                    </mat-panel-title>
                </mat-expansion-panel-header>
                <div class="px-3">
                    Each ANANASTRA report page contains 3 sections: (1) Summary, (2) ASB events related to particular TFs or cell types, and (3) Complete table of ASBs found at user-submitted SNPs.
                    <dl>
                        <dt>Summary</dt>
                        <dd>
                            This section is dedicated to general information on ASB events found in the submitted set of SNPs.
                            Particularly, we report the observed fraction of ASBs
                            (among all candidate ASB sites passing certain read coverage thresholds, see <>)
                            ASB enrichment significance is calculated with the right-tail Fisher's exact test, see
                            <a
                                (click)="openGlossary('ASB_enrichment'); $event.preventDefault()"
                                href="/help#ASB_enrichement"
                            >
                                here</a>.
                        </dd>
                        <dt>ASB events related to particular TFs or cell types</dt>
                        <dd>The second section contains information about ASB events of particular
                        TFs or cell types. For each TF and cell type, we compute a separate P-value
                        using the same strategy as for the global enrichment, the P-values are then
                        corrected for the number of tested TFs / cell types (with at least 1 observed
                        ASB in the user's list) using Benjamini-Hogberg (FDR) procedure.
                        The TFs / cell types with the lowest FDR are displayed on top by default.</dd>
                        <dt>Complete table of ASBs found at user-submitted SNPs</dt>
                        <dd>
                            The final section provides a detailed table listing all ASB events
                            (in the expanded view mode) or showing only the single most
                            significant TF-/Cell type- ASB event per SNP (in the collapsed view mode).
                            Each mode allows downloading a complete table (the list of columns visible
                            in the online report can be adjusted when necessary).
                        </dd>
                    </dl>
                </div>
            </mat-expansion-panel>
        </mat-accordion>
    </div>
</div>
